### ISSUE TITLE
R2gen MIMIC image

### ISSUE THREAD

**Issue by nobottle** [jhb86253817/PromptMRG#4]
I read in the article that you used R2gen MIMIC images, are they different in shape or something from the MIMIC images provided by physionet?

**Comment by jhb86253817** [comment]
Yes, I think the images from R2Gen has been processed, which has shape 256x256 while the ones from physionet are original images. Even if you resize the original images to 256, they may still be slightly different in pixel values.

**Comment by nobottle** [comment]
Thank you, that was very helpful. Just out of curiosity, I asked for a mimic image for R2gen, but got no answer, so I'll have to utilize the physionet original image...

**Comment by afasijbfk** [comment]
> Thank you, that was very helpful. Just out of curiosity, I asked for a mimic image for R2gen, but got no answer, so I'll have to utilize the physionet original image...
Is there any difference between the results you got with the physionet original image and the R2gen image results? Thank you

**Comment by afasijbfk** [comment]
> Thank you, that was very helpful. Just out of curiosity, I asked for a mimic image for R2gen, but got no answer, so I'll have to utilize the physionet original image...

Is there any difference between the results you got with the physionet original image and the R2gen image results? Thank you very much

**Comment by jhb86253817** [comment]
> > Thank you, that was very helpful. Just out of curiosity, I asked for a mimic image for R2gen, but got no answer, so I'll have to utilize the physionet original image...
> 
> Is there any difference between the results you got with the physionet original image and the R2gen image results? Thank you very much

As long as you train and test on the same image source, the results of the two are similar.
